File: INF.04-01-21.06-SG/konsola/konsola.cpp
#include <iostream>
using namespace std;

class tablica {
    private:
        int tab[10];

        int znajdzNajwieksza()
        {
            int najwiekszaLiczba = tab[0];

            for (int i = 0; i < 10; i++) 
            {
                if (tab[i] > tab[najwiekszaLiczba])
                {
                    najwiekszaLiczba = i;
                }    
            }

            return tab[najwiekszaLiczba];
        }
    public:

        void wczytaj()
        {
            cout << "Wypisz 10 liczb calkowitych: \n";
            for (int i = 0; i < 10; i++)
            {
                cin >> tab[i];
            }
            cout << "\nNajwieksza liczba z wypisanych jest: " << znajdzNajwieksza() << "\n";
        }

        void sortuj()
        {
            for (int i = 0; i < 10; i++) {
                for (int j = i; j < 10; j++) {
                    if (tab[i] > tab[j]) {
                        swap(tab[i], tab[j]);
                    }
                }
            }

            cout << "Posortowana tablica: \n";
            for (int i = 0; i < 10; i++)
                cout << tab[i] << "\n";
        }
};

int main()
{
    tablica t1{};

    t1.wczytaj();
    
    t1.sortuj();

    return 1;   
}

File: INF.04-01-21.06-SG/konsola/konsola.cs
using System;

class Tablica
{
    private int[] tab = new int[10];

    private int ZnajdzNajwieksza()
    {
        int najwiekszaLiczba = tab[0];

        for (int i = 1; i < 10; i++)
        {
            if (tab[i] > najwiekszaLiczba)
            {
                najwiekszaLiczba = tab[i];
            }
        }
        return najwiekszaLiczba;
    }

    public void Wczytaj()
    {
        Console.WriteLine("Wypisz 10 liczb calkowitych:");
        for (int i = 0; i < 10; i++)
        {
            tab[i] = int.Parse(Console.ReadLine());
        }
        Console.WriteLine($"\nNajwieksza liczba z wypisanych jest: {ZnajdzNajwieksza()}\n");
    }

    public void Sortuj()
    {
        for (int i = 0; i < 10; i++)
        {
            for (int j = i + 1; j < 10; j++)
            {
                if (tab[i] > tab[j])
                {
                    int temp = tab[i];
                    tab[i] = tab[j];
                    tab[j] = temp;
                }
            }
        }

        Console.WriteLine("Posortowana tablica:");
        foreach (int liczba in tab)
        {
            Console.WriteLine(liczba);
        }
    }
}

class Program
{
    static void Main()
    {
        Tablica t1 = new Tablica();

        t1.Wczytaj();
        t1.Sortuj();
    }
}